app/assets/stylesheets/govuk_elements_extensions.scss

```diff
@@ -176,13 +173,14 @@ section {
   border: 1px solid $grey-2;
 }
 
+input[disabled],
 input[disabled="disabled"],
 input[readonly="readonly"],
 textarea[disabled],
 textarea[disabled="disabled"],
 textarea[readonly="readonly"] {
   opacity: 1;
-  background-color: $grey-4;
+  background-color: #eee;
```

Comment:
Ok moving to a colour variable. At present we have not defined any other colour variables so leaving it in the same scss file for now.